docs/overview.md

```diff
@@ -145,7 +149,7 @@ Sphinx and MyST take very different approaches to publishing to the web. In Sphi
 
 Although the official engine also supports static HTML outputs, implementing themes as applications makes it easier to build more powerful MyST viewing experiences.
 
-[^api-docs]: In the future, `mystmd` may offer support for Python and JavaScript documentation, and if you want to contribute please reach out!
+[^api-docs]: In the future, `mystmd` may offer support for Python and JavaScript documentation, and if you want to contribute please discuss on [#1259](https://github.com/jupyter-book/mystmd/issues/1259) and this [video](https://www.loom.com/share/589a947389fb49829c9b712b796d8a7d).
```

Comment:
Added a few more links and closed some issues to direct comments to the parent issue.